hackathon_site/dashboard/frontend/src/components/inventory/Item/Item.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import styles from "./Item.module.scss";
+import Card from "@material-ui/core/Card";
+import CardMedia from "@material-ui/core/CardMedia";
+import FiberManualRecordIcon from "@material-ui/icons/FiberManualRecord";
+import Grid from "@material-ui/core/Grid";
+
+const Item = ({ image, title, total, limit, currentStock }) => {
```

Comment:
just remove the limit prop entirely, I think it will be handled by Redux in Aki's component